### ISSUE TITLE
Get warning upon `using FlexibleSpacecraft`

### ISSUE THREAD

**Issue by Mizu49** [Mizu49/FlexibleSpacecraft.jl#98]
When importing `FlexibleSpacecraft` module, we got the following warning: 

```text
┌ Warning: Type annotations on keyword arguments not currently supported in recipes. Type information has been discarded
└ @ RecipesBase ~/.julia/packages/RecipesBase/qpxEX/src/RecipesBase.jl:123
┌ Warning: Type annotations on keyword arguments not currently supported in recipes. Type information has been discarded
└ @ RecipesBase ~/.julia/packages/RecipesBase/qpxEX/src/RecipesBase.jl:123
```

This might be an unsupported type annotation on keyword arguments in plot recipes. Fix this.

**Comment by Mizu49** [comment]
Removing type annotation in `@recipe` gives another error...

```julia
julia> fig3 = PlotRecipe.framegif(time, LVLHref, attitudedata.rollpitchyawframe, Tgif = 20, FPS = 8)
ERROR: type Array has no field step
Stacktrace:
  [1] getproperty(x::Vector{Int64}, f::Symbol)
    @ Base ./Base.jl:42
  [2] macro expansion
    @ /workspaces/FlexibleSpacecraft.jl/src/PlotRecipes/PhysicalQuantity.jl:86 [inlined]
  [3] apply_recipe(plotattributes::AbstractDict{Symbol, Any}, time::Any, angularvelocity::Any, axisindex::Any)
    @ Main.FlexibleSpacecraft.PlotRecipe.PhysicalQuantity ~/.julia/packages/RecipesBase/qpxEX/src/RecipesBase.jl:289
  [4] _process_userrecipes!(plt::Any, plotattributes::Any, args::Any)
    @ RecipesPipeline ~/.julia/packages/RecipesPipeline/F2mWY/src/user_recipe.jl:36
  [5] recipe_pipeline!(plt::Any, plotattributes::Any, args::Any)
    @ RecipesPipeline ~/.julia/packages/RecipesPipeline/F2mWY/src/RecipesPipeline.jl:70
  [6] _plot!(plt::Plots.Plot, plotattributes::Any, args::Any)
    @ Plots ~/.julia/packages/Plots/rDbWY/src/plot.jl:208
  [7] #plot#139
    @ ~/.julia/packages/Plots/rDbWY/src/plot.jl:91 [inlined]
  [8] quiver(::Any, ::Vararg{Any}; kw::Base.Pairs{Symbol, V, Tuple{Vararg{Symbol, N}}, NamedTuple{names, T}} where {V, N, names, T<:Tuple{Vararg{Any, N}}})
    @ Plots ~/.julia/packages/RecipesBase/qpxEX/src/RecipesBase.jl:410
  [9] dispframe(time::Float64, refCoordinate::Frame, coordinate::Frame)
    @ Main.FlexibleSpacecraft.PlotRecipe.FramePlot /workspaces/FlexibleSpacecraft.jl/src/PlotRecipes/FramePlot.jl:22
 [10] macro expansion
    @ /workspaces/FlexibleSpacecraft.jl/src/PlotRecipes/FramePlot.jl:110 [inlined]
 [11] macro expansion
    @ ~/.julia/packages/Plots/rDbWY/src/animation.jl:197 [inlined]
 [12] framegif(time::StepRangeLen{Float64, Base.TwicePrecision{Float64}, Base.TwicePrecision{Float64}, Int64}, refframe::Frame, frames::StructArrays.StructVector{Frame, NamedTuple{(:x, :y, :z), Tuple{Vector{StaticArrays.SVector{3, Real}}, Vector{StaticArrays.SVector{3, Real}}, Vector{StaticArrays.SVector{3, Real}}}}, Int64}; Tgif::Int64, FPS::Int64, timerange::Tuple{Int64, Int64})
    @ Main.FlexibleSpacecraft.PlotRecipe.FramePlot /workspaces/FlexibleSpacecraft.jl/src/PlotRecipes/FramePlot.jl:108
 [13] top-level scope
    @ REPL[17]:1
```

**Comment by Mizu49** [comment]
As is shown in the warning message, the type annotation of the keyword arguments for the function with the `@recipe` macro is not supported. Removing the type annotation for the keyword argument resolved the bug.

```julia
# this is not working
@recipe function f( ..., datalabel::String = "") 

# this is OK
@recipe function f( ..., datalabel = "")
```

**Comment by Mizu49** [comment]
This issue is closed now #111
